wsd/apps/rest/v0/views/notification.py

```diff
@@ -21,3 +25,29 @@ class NotificationViewSet(BaseModelViewSet):
     def get_queryset(self):
         user = self.request.user
         return self.model.objects.filter(user=user) if user.is_authenticated else self.model.objects.none()
+
+    @extend_schema(
+        summary="Mark all notifications as read",
+        description="Marks all notifications for the current user as read",
+        responses={
+            200: {"type": "object", "properties": {"message": {"type": "string"}}},
+            401: None,
+            403: {"type": "object", "properties": {"detail": {"type": "string"}}},
+        },
+    )
+    @action(detail=False, methods=["post"])
```

Comment:
Another nitpick: Can you add `url_path="mark-all-as-read"` so we have "prettier" urls, dashes instead of underscores

Reply:
Ok, i thought about it as _ are more "common" in other projects thus i chose that but will change it.